### PR TITLE
replaces @internalProperty() to @state

### PR DESCRIPTION
**Pull request by gisumit**
With new "Lit" dependency.... the `internalProperty` decorater is renamed to `state` and bumped the package version.

### CODE REVIEW

File: packages/opc-comment-input/src/opc-comment-input.ts [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 @customElement('opc-comment-input')
 export class CommentInput extends LitElement {
   @property({type: String})
   placeholder = 'Type your comment';
 
-  @internalProperty()
+  @state()
   protected commentText = '';
 
   @query('textarea')
```
Comment (hybridx) [inline packages/opc-comment-input/src/opc-comment-input.ts:13]: Uses this from `lit/decorators.js` as well. Don't use `iit-element`

Comment (hybridx) [reply packages/opc-comment-input/src/opc-comment-input.ts:13]:
```
import {html, css, LitElement} from 'lit';
import {customElement, property} from 'lit/decorators.js';

```

**Review by hybridx — DISMISSED** [review]

File: packages/opc-comment-input/src/opc-comment-input.ts [hunk 2]
```diff
@@ -1,12 +1,13 @@
-import { LitElement, html, property, customElement, query, internalProperty } from 'lit-element';
+import { LitElement, html, property, customElement, query } from 'lit-element';
```
Comment (riginoommen) [inline packages/opc-comment-input/src/opc-comment-input.ts:1]: import { html, LitElement } from 'lit';
import {customElement, property, state, query }  from 'lit/decorators.js';

**Review by riginoommen — CHANGES_REQUESTED** [review]

File: packages/opc-comment-input/src/opc-comment-input.ts (hunk 1 above)
Comment (gisumit) [reply packages/opc-comment-input/src/opc-comment-input.ts:13]: yeah, updated the imports.

**Review by riginoommen — DISMISSED** [review]
Bump the version for the fix

**Review by riginoommen — APPROVED** [review]

**Review by hybridx — APPROVED** [review]

**Pull request merged**